app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/welcomescreen/WelcomeScreen.kt

```diff
@@ -0,0 +1,152 @@
+import hr.dice.filipbionda.tmdbpractice.ui.theme.welcomeScreenPrimaryTextColor
+
+@Composable
+fun WelcomeScreen(modifier: Modifier = Modifier)  {
+    Box(
+        modifier = modifier.fillMaxSize(),
+    ) {
+        val brush =
+            Brush.verticalGradient(
+                colorStops =
+                    arrayOf(
+                        0.0f to overlayGradientFirstColor,
+                        0.8f to overlayGradientSecondColor,
+                    ),
+            )
+        Image(
+            painter = painterResource(R.drawable.welcomescreen_background_blurred),
+            contentDescription = null,
+            modifier =
+                Modifier
+                    .fillMaxSize(),
+            contentScale = ContentScale.FillBounds,
+        )
+        Canvas(
+            modifier = Modifier.fillMaxSize(),
+        ) {
+            drawRect(
+                brush = brush,
+            )
+        }
+        Column(
+            modifier =
+                Modifier.align(alignment = Alignment.BottomCenter)
+                    .padding(horizontal = 28.dp, vertical = 42.dp),
+        ) {
+            val annotatedString =
+                buildAnnotatedString {
+                    withStyle(
+                        style = SpanStyle(color = welcomeScreenPrimaryTextColor),
+                    ) {
+                        append(stringResource(R.string.welcome_screen_text_part1))
+                    }
+                    withStyle(
+                        style =
+                            SpanStyle(
+                                color = welcomeScreenPrimaryTextColor,
+                                fontWeight = FontWeight.Bold,
+                            ),
+                    ) {
+                        append(stringResource(R.string.welcome_screen_text_part2))
+                    }
+                }
+            val buttonHorizontalGradientBrush =
+                Brush.horizontalGradient(
+                    colors = listOf(buttonFirstColor, buttonSecondColor),
+                )
+            Image(
+                painter = painterResource(R.drawable.img),
+                contentDescription = null,
+                contentScale = ContentScale.FillBounds,
+                modifier =
+                    Modifier
+                        .width(176.dp)
+                        .height(18.dp),
+            )
+            Spacer(
+                modifier = Modifier.height(20.dp),
+            )
+            Text(
+                text = annotatedString,
+                color = welcomeScreenPrimaryTextColor,
+                fontSize = 30.sp,
+                lineHeight = 36.sp,
+            )
+            Spacer(
+                modifier = Modifier.height(20.dp),
+            )
+            Text(
+                text = stringResource(R.string.welcome_screen_description),
+                color = welcomeScreenPrimaryTextColor,
```

Comment:
This text is missing `style` from the typography that you need to create and pass from the theme

Reply:
done